torch/csrc/jit/script/compiler.cpp

```diff
@@ -160,6 +160,11 @@ struct Environment {
       // Actually remove the input
       b->eraseInput(*ritr);
       captured_inputs.erase(captured_inputs.begin() + *ritr - skip_num);
+      for (const auto& kv : value_table) {
+        if (kv.second == v) {
+          value_table[kv.first] = orig;
+        }
+      }
```

Comment:
This can't be `value_table[name] = orig` because we process `q = x` (with an input `x`) by only setting `value_map["q"] = value_map["x"]`? Is that right?